testdata/expected/src/github.com/sgtest/minimal-go-stdlib/builtin/GoPackage.unit.json

```diff
@@ -1 +1 @@
-{"Name":"builtin","Type":"GoPackage","Files":["src/builtin/builtin.go"],"Dir":"src/builtin","Data":{"Dir":"src/builtin","Name":"builtin","ImportComment":"","Doc":"","ImportPath":"builtin","Root":"","SrcRoot":"","PkgRoot":"","PkgTargetRoot":"","BinDir":"","Goroot":true,"PkgObj":"","AllTags":null,"ConflictDir":"","GoFiles":["builtin.go"],"CgoFiles":null,"IgnoredGoFiles":null,"InvalidGoFiles":null,"CFiles":null,"CXXFiles":null,"MFiles":null,"HFiles":null,"SFiles":null,"SwigFiles":null,"SwigCXXFiles":null,"SysoFiles":null,"CgoCFLAGS":null,"CgoCPPFLAGS":null,"CgoCXXFLAGS":null,"CgoLDFLAGS":null,"CgoPkgConfig":null,"Imports":[],"ImportPos":null,"TestGoFiles":null,"TestImports":[],"TestImportPos":null,"XTestGoFiles":null,"XTestImports":[],"XTestImportPos":null},"Config":{"GOROOT":"."},"Ops":{"depresolve":null,"graph-all":null}}
```

Comment:
I wish github could make it easy for me to see that just `"Config":{"GOROOT":"."}` had been removed. I assume that is the only difference?